.gitignore

```diff
@@ -66,6 +66,7 @@ doc/source/docs/reference/api/*.rst
 
 geopandas.egg-info
 geopandas/version.py
+geopandas/datasets/ne_110m_admin_0_countries.zip
```

Comment:
This is not related to this PR but to the #2418 so it should not be here.

Reply:
done - got rid of file.  hopefully I don't have to download again for other PR